installing/installing_vsphere/installing-vsphere-assisted-installer.adoc

```diff
@@ -0,0 +1,21 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="installing-vsphere-assisted-installer"]
+= Installing a cluster on vSphere using the Assisted Installer
+include::_attributes/common-attributes.adoc[]
+:context: installing-vsphere-assisted-installer
+
+toc::[]
+
+You can install {product-title} on on-premise hardware or on-premise VMs by using the {ai-full}. Installing {product-title} by using the {ai-full} supports `x86_64`, `AArch64`, `ppc64le`, and `s390x` CPU architectures.
+
+The {ai-full} is a user-friendly installation solution offered on the Red Hat Hybrid Cloud Console. The {ai-full} supports the various deployment platforms with a focus on the following infrastructures:
```

Comment:
Tiny nit that I just learned last week: apparently we should use a `{nbsp}` in between Red and Hat so the company name doesn't get broken up onto two separate lines, per the [RHSSG](https://redhat-documentation.github.io/supplementary-style-guide/#non-breaking-spaces)

```suggestion
The {ai-full} is a user-friendly installation solution offered on the Red{nbsp}Hat Hybrid Cloud Console. The {ai-full} supports the various deployment platforms with a focus on the following infrastructures:
```